build_projects/dotnet-cli-build/project.json

```diff
@@ -13,7 +13,7 @@
     "System.Runtime.Serialization.Primitives": "4.1.1-rc3-24131-00",
     "System.Xml.XmlSerializer": "4.0.11-rc3-24131-00",
     "WindowsAzure.Storage": "6.2.2-preview",
-    "NuGet.CommandLine.XPlat": "3.5.0-rc-1285",
+    "NuGet.CommandLine.XPlat": "3.5.0-beta2-1392",
```

Comment:
As far as I know, we made a mistake in the build system where beta packages published to the myget feed were labelled as RC.

@rrelyea  might have more context on this.